frontend/chains/akash.json

```diff
@@ -21,8 +21,8 @@
   "config": {
     "chain_id": "akashnet-2",
     "chain_name": "Akash",
-    "rest": "https://api.resolute.vitwit.com/akash_api/",
-    "rpc": "https://api.resolute.vitwit.com/akash_rpc/",
+    "rest": "https://api.resolute.vitwit.com/akash_api",
```

Comment:
This should be handled at the application level by performing validation. If there is `/` at the end, we should remove it.

Reply:
Okay aleem